app/models/project.rb

```diff
@@ -116,4 +116,8 @@ def plugin_uploads_node
   def recovered
     @recovered ||= nodes.find_or_create_by(label: 'Recovered')
   end
+
+  def report_template_properties
+    nil
+  end
```

Comment:
Feels like #weaksauce but perhaps is #weaksauce we have to live with.

Reply:
The alternative to this is explicit edition checks with `defined?`. While that is what we do with some parts of the app, this is way simpler and something I think we can live with.